cli/dts/lib.deno.ns.d.ts

```diff
@@ -2953,6 +2953,7 @@ declare namespace Deno {
     | "ANAME"
     | "CNAME"
     | "MX"
+    | "NS"
```

Comment:
A new overload for `resolveDns` should be added to this file as well

Reply:
Actually, a new overload is probably not needed, but "NS" variant should be added to one of existing overloads.

Reply:
resolved in 6ccb2ce03ca1ffe3d298dfda028124525d7848a4